hudi-utilities/src/main/java/org/apache/hudi/utilities/UtilHelpers.java

```diff
@@ -578,6 +580,20 @@ public static HoodieTableMetaClient createMetaClient(
         .build();
   }
 
+  public static void addLockOptions(String basePath, TypedProperties props) {
+    if (!props.containsKey(HoodieLockConfig.LOCK_PROVIDER_CLASS_NAME.key())) {
+      HoodieLockConfig lockConfig = HoodieLockConfig.newBuilder()
+          .withLockProvider(FileSystemBasedLockProvider.class)
+          .withLockWaitTimeInMillis(2000L) // 2s
```

Comment:
Could the user only config the option like `hoodie.write.lock.wait_time_ms_between_retry` etc?

Reply:
Probably, the first step is to support a default config option.